docs/source/usage/01_usage.md

```diff
@@ -68,6 +68,7 @@ are shared among multiple roles:
 - `cifmw_nolog`: (Bool) Toggle `no_log` value for selected tasks. Defaults to `true` (hiding those logs by default).
 - `cifmw_parent_scenario`: (String or List(String)) path to existing scenario/parameter file to inherit from.
 - `cifmw_configure_switches`: (Bool) Specifies whether switches should be configured. Computes in `reproducer.yml` playbook. Defaults to `false`.
+- `cifmw_ntp_server`: (String) Specifies an ntp server to use. Now it's only defined in `controller-0` environment. Defaults to `clock.redhat.com`.
```

Comment:
please change the default to a public server

Reply:
done